package.json

```
@@ -69,5 +69,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

```suggestion
  }
```

Remove auto-pinned wrong packageManager

Set environment variable [COREPACK_ENABLE_AUTO_PIN=0](https://github.com/nodejs/corepack/blob/main/README.md#environment-variables) in local profile to avoid this happening again.